templates/firefox-ios.xliff

```diff
@@ -3062,6 +3056,29 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Shared/Supporting Files/Today.strings" source-language="en" datatype="plaintext">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
+    </header>
+    <body>
+      <trans-unit id="TodayWidget.CopiedLinkLabelFromPasteBoardV1">
+        <source>Copied Link from clipboard</source>
+        <note>Copied Link from clipboard displayed</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.GoToCopiedLinkLabelV1">
+        <source>Go to copied link</source>
+        <note>Go to link pasted on the clipboard</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.NewTabButtonLabelV1">
+        <source>New Search</source>
+        <note>Open New Tab button label</note>
```

Comment:
Should the localization note here be updated to "Open New Search button label"?

Reply:
Ah, basically it is opening new tab but we present it as new search as the address bar is where user searches. 

Would you suggest updating? 🤔

Reply:
Ah gotcha! No, I think it should be ok as is then. Thank you!